src/main/java/no/unit/nva/cristin/projects/NvaProjectBuilder.java

```diff
@@ -93,24 +100,17 @@ private static NvaPerson mapCristinPersonToNvaPerson(Person person) {
 
     private static NvaOrganization mapCristinInstitutionToNvaOrganization(Institution institution) {
```

Comment:
this method belongs to NvaOrganziaiton og Insittuion

Reply:
Can you elaborate? Do you want med to put logic in model classes?